addons/mass_mailing/static/src/views/fields/mass_mailing_percent_pie_field.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { percentPieField, PercentPieField } from "@web/views/fields/percent_pie/percent_pie_field";
+
+export class MailingPercentPieField extends  PercentPieField{
```

Comment:
I don't think we are going to add JS code just to display 25.00 instead of 25... honestly that's overengineering ^^ .